src/main/java/com/snowflake/kafka/connector/internal/BufferThreshold.java

```diff
@@ -256,9 +247,9 @@ private static ImmutableMap<String, String> verifyBufferCountThreshold(
   @Override
   public String toString() {
```

Comment:
umm, can you verify if this class is serialized somewhere?

Reply:
like in telemetry? 
If yes, can you call that out as well..

Reply:
good point. BufferThreshold is only logged and serialized in a log message indicating that flush has occurred. I've updated the description with that information

Reply:
The buffer thresholds are also logged in telemetry, however it is logged as part of the larger Map<String, String> config, rathe than as a BufferThreshold which would be serialized here

Reply:
Can you make changes to the wiki regarding the same! Where we write helpful debug queries. 
For instance we will have two types of telemetries based on versions. 

Thanks